packages/wagmi-core/src/actions/sendEmailOtp.ts

```diff
@@ -22,5 +22,6 @@ export async function sendEmailOtp(
   const connector = resolveAlchemyAuthConnector(config);
 
   const authClient = connector.getAuthClient();
+  // TODO(v5): this fails if the account doesn't exist yet. do we want to handle creating accounts here?
```

Comment:
i thought that's a bug with the signer/auth implementation, and that a new account should automatically be created since we don't have a separate flow for new accounts. @dphilipson is that correct?